docs/recipes.md

```diff
@@ -589,4 +589,31 @@ it has not seen, it will compute the predictions and store them in a cache. The
 next time it is called with the same parameters, instead of recomputing the
 predictions, it will return the previously computed value from the cache.
 
+Unlike `functools.cache`, `mo.cache` is notebook aware; and will become
+invalidated if a relevant global variable is changed (including State and UI
+Elements), or the function definition is changed.
```

Comment:
"or the function definition is changed" -- won't this be true for `functools.cache` too?

Reply:
"but not if the cell is rerun with no changes"

But will workshop this sentence